CONTRIBUTION.md

```diff
@@ -0,0 +1,78 @@
+# Contributing to MSTS Workshops
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to MSTS Workshop curriculum. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Workshop Goals](#goals)
+  * [Constraints](#constraints)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Suggest an Enhancement](#suggest-an-enhancement)
+  * [Build a new session plan](#build-a-plan)
+  * [Pull Requests](#pull-requests)
+
+[Session Templates](#session-templates)
+  * [Session Template](#session-template)
+
+[Additional Notes](#additional-notes)
+
+## What should I know before I get started?
+
+The contribution should be aligned with the workshop goals, and conform to the constraints.
+
+### Workshop Goals
+1. Introduce people to tools and methods that aren’t used here to foster curiosity
+1. Help people establish cross-team working relationships in low risk, low pressure environments
+1. Serve as a way for people to disconnect from their daily, muddled problems and think on something more constrained
+
+
+### Constraints
+* 30 minute session
+* No Pre-work
+* No Post-work
+
+## How can I contribute?
+Workshops are always changing. As the audience changes and grows, the material to needs to adjust to stay relevant. If you feel like something needs to change, change it.
+
+1. Fork the project
+1. Make the change on your repository
+1. Submit a Pull Request to the MSTS/Workshops repository
+1. Participate in the discussion that will form around the Pull Request
+
+### Suggest an enhancement
+The existing lessons are starting points to facilitate discussion. If you feel like additional information would make a workshop session better. Make the change you would like to see.
```

Comment:
Was this supposed to read "...session better, make..."?

Reply:
fixed!